### ISSUE TITLE
green-yellow line flashing

### ISSUE THREAD

**Issue by fjf2002** [martin-pabst/Online-IDE-new-compiler#39]
Press F5 very often on a very short program.

Every 10th time or so you will see the first code line flash in green-yellow.

A bug, I think? When executing with F5 (i. e. no steps), no green-yellow is neccessary !?

**Comment by Johannes-Pabst** [comment]
As far as I can tell, this isn't a bug. I assume you where trying this with code like `new World();` which takes more time than something like `println("test");` because this was the only way I found to reproduce the issue. If this is the case, the "green-yellow line" should be the instruction pointer. Normally, when running code at maximum speed, this shouldn't be visible, but as far as I can tell in rare cases, the program is started (first F5 stroke), running, and then paused (second F5 stroke) and resumed (third F5-stroke) before it was completed which caused the flickering yellow-green line.
The F5-key is also used to pause the programm while it's running and to resume execution when it's paused. 

I also tried holding down and then releasing the F5-key repeatedly. This will sometimes result in a paused programm, which, as far as I can tell, confirms my theory.
